src/timezone.ts

```diff
@@ -18,12 +18,13 @@ export const timezone = async (message: Message) => {
     }
 
     const match = message.content.match(/([\d]{1,2})([:\d]{3})?[\s]?(pm|am|AM|PM)?\b[\s]([a-zA-Z]{2,3})?\b/);
+    
```

Comment:
Please remove spaces on line 21 & 27 😁

Reply:
Thanks for checking on this. Working on it after noticing the it wasn't passing linting.